components/request_otr/browser/request_otr_component_installer.cc

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/request_otr/browser/request_otr_component_installer.h"
+
+#include <memory>
+#include <utility>
+
+#include "base/base_paths.h"
+#include "base/command_line.h"
+#include "base/containers/flat_set.h"
+#include "base/json/json_reader.h"
+#include "base/logging.h"
+#include "base/task/thread_pool.h"
+#include "base/types/expected.h"
+#include "brave/components/brave_component_updater/browser/dat_file_util.h"
+#include "brave/components/brave_component_updater/browser/local_data_files_service.h"
+#include "net/base/registry_controlled_domains/registry_controlled_domain.h"
+
+using brave_component_updater::LocalDataFilesObserver;
+using brave_component_updater::LocalDataFilesService;
+
+namespace request_otr {
+
+const char kRequestOTRConfigFile[] = "request-otr.json";
+const char kRequestOTRConfigFileVersion[] = "1";
+
+RequestOTRComponentInstaller::RequestOTRComponentInstaller(
```

Comment:
oddly the class naming convention for these are `*ComponentInstallerPolicy`, but the file name is correct